Packs/GoogleDrive/ReleaseNotes/1_2_29.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Google Drive
+
+- Added the **include_items_from_all_drives** argument to **google-drive-files-list** and **google-drive-files-list** commands
```

Comment:
- Added the **include_items_from_all_drives** argument to **google-drive-files-list** and **google-drive-file-get** commands